app/views/pages/home.html.erb

```diff
@@ -1 +1,44 @@
-<h2>Welcome to Classroom!</h2>
+<div class="signin-section">
+  <div class="container text-center">
+    <ul>
+      <li><%= mega_octicon('book')%></li>
+      <li><%= mega_octicon('pencil')%></li>
+      <li><%= mega_octicon('mortar-board')%></li>
+      <li><%= mega_octicon('organization')%></li>
+      <li><%= mega_octicon('calendar')%></li>
+      <li><%= mega_octicon('comment-discussion')%></li>
+    </ul>
+
+    <h1 class="heading">Your course assignments on GitHub</h1>
+    <p class="subheading">Automate repository creation and access control, making it easy to distribute starter code and collect assignment submissions on GitHub.</p>
```

Comment:
Let's use "Classroom for GitHub automates repository creation and access control, making it easy to distribute starter code and collect assignments on GitHub."

We can probably do better, but what's currently there doesn't seem correct.